storage/ndb/rest-server/rest-api-server/internal/handlers/batchpkread/handler.go

```diff
@@ -127,7 +127,7 @@ func (h *Handler) Execute(request interface{}, response interface{}) (int, error
 }
 
 func processResponses(respBuffs *[]*heap.NativeBuffer, response api.BatchOpResponse) (int, error) {
-	for _, respBuff := range *respBuffs {
+	for idx, respBuff := range *respBuffs {
```

Comment:
Does this imply that idx gets incremented in the loop?

Reply:
Yes